package.json

```diff
@@ -47,10 +47,10 @@
     "mariadb": "^2.5.3",
     "morgan": "^1.10.0",
     "mysql2": "^2.2.5",
-    "pg": "^8.5.1",
     "pg-hstore": "^2.3.2",
     "sequelize": "^6.6.2",
     "short-unique-id": "^4.4.4",
+    "sqlite3": "^5.0.2",
```

Comment:
Don't change the package dependencies, please

Reply:
Please also remove the package-lock.json from the changes.